code/game/machinery/navbeacon.dm

```diff
@@ -1,6 +1,15 @@
 // Navigation beacon for AI robots
 // No longer exists on the radio controller, it is managed by a global list.
 
+/obj/item/navbeacon
```

Comment:
Don't forget to add comments at the start and end of each non-modular edited section so we don't accidentally write over it later during updates
Just something like //MonkeStation Edit Start/End

Reply:
Done!